### ISSUE TITLE
off-by-one in exp verification

### ISSUE THREAD

**Issue by vsespb** [DCIT/perl-Crypt-JWT#13]
```
croak "JWT: exp claim check failed ($payload->{exp}/$leeway vs.
$now)" if $payload->{exp} + $leeway < $now;
```

if we assume leeway ==0, then if exp == now, it will not raise expection.

however RFC tells it must:

```
===

The "exp" (expiration time) claim identifies the expiration time on
   or after which the JWT MUST NOT be accepted for processing.  The
   processing of the "exp" claim requires that the current date/time
   MUST be before the expiration date/time listed in the "exp" claim
===
```

**Comment by karel-m** [comment]
Do you mean changing it like:
```diff
- croak "..." if $payload->{exp} + $leeway < $now;
+ croak "..." if $payload->{exp} + $leeway <= $now;
```
?

**Comment by vsespb** [comment]
yes

**Comment by karel-m** [comment]
Should be fixed in Crypt-JWT-0.021

Thanks for your time.
